packages/core/src/Services/socket-general.js

```diff
@@ -159,13 +159,7 @@ const BinarySocketGeneral = (() => {
                     if (window.TrackJS) window.TrackJS.track('Custom InvalidToken error');
                 }
                 client_store.logout().then(() => {
-                    common_store.setError(true, {
-                        header: response.error.message,
-                        message: localize('Please Log in'),
-                        should_show_refresh: false,
-                        redirect_label: localize('Log in'),
-                        redirectOnClick: () => redirectToLogin(false, getLanguage()),
-                    });
+                    common_store.routeTo(routes.trade);
```

Comment:
what if they're using a platform other than `DTrader`? 🤔

Reply:
Actually this is the expected behavior requested in the card. Let me check this to make sure.